common/messaging/kafka/clientImpl.go

```diff
@@ -86,16 +86,20 @@ func NewKafkaClient(
 // NewConsumer is used to create a Kafka consumer
 func (c *clientImpl) NewConsumer(app, consumerName string) (messaging.Consumer, error) {
 	topics := c.config.GetTopicsForApplication(app)
+	version, err := sarama.ParseKafkaVersion(c.config.GetKafkaVersionForApplication(app))
+	if err != nil {
+		return nil, err
+	}
 	saramaConfig := sarama.NewConfig()
 	// bellow config is copied from uber/kafka-clientImpl bo keep the same behavior
-	saramaConfig.Version = sarama.V0_10_2_0
+	saramaConfig.Version = version
```

Comment:
do we presume to support the all kafka versions here?

Reply:
I think so, unless there is a reason not to do so.